js/ui/shellDBus.js

```diff
@@ -606,3 +611,80 @@ var HackableAppsManager = class {
         return paths;
     }
 }
+
+const AppLauncherIface = '<node> \
+<interface name="org.gnome.Shell.AppLauncher"> \
+<method name="Launch"> \
+    <arg type="s" direction="in" name="name" /> \
+    <arg type="u" direction="in" name="timestamp" /> \
+</method> \
+<method name="LaunchViaDBusCall"> \
+    <arg type="s" direction="in" name="name" /> \
+    <arg type="s" direction="in" name="busName" /> \
+    <arg type="s" direction="in" name="objectPath" /> \
+    <arg type="s" direction="in" name="interfaceName" /> \
+    <arg type="s" direction="in" name="methodName" /> \
+    <arg type="v" direction="in" name="args" /> \
+</method> \
+</interface> \
+</node>';
```

Comment:
Please move this to under `data/dbus-interfaces/org.gnome.Shell.AppLauncher.xml` and load it with `loadInterfaceXML('org.gnome.Shell.AppLauncher')`